### PR TITLE
REL-2155 Update SonarQube to 8.9.7

### PR DESCRIPTION
**Pull request by philippe-perrin-sonarsource**

### CODE REVIEW

File: charts/sonarqube-lts/Chart.yaml [hunk 1]
```diff
@@ -18,9 +18,11 @@ maintainers:
     email: tobias.trabelsi+helm@sonarsource.com
 annotations:
   artifacthub.io/changes: |
+    - kind: changed
+      description: "updated SonarQube to 8.9.7"
     - kind: fixed
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube-lts/Chart.yaml:23]: The previous fix can be removed from the list. It is already live on [artifacthub](https://artifacthub.io/packages/helm/sonarqube/sonarqube-lts?modal=changelog)

**Review by tobias-trabelsi-sonarsource — COMMENTED** [review]
QA can only be green when images are available

**Comment by philippe-perrin-sonarsource** [comment]
> QA can only be green when images are available

Interresting, this isn't what's stated in the doc. I'll update it.

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]
images are available, so just the annotation left

**Review by tobias-trabelsi-sonarsource — APPROVED** [review]
LGTM :+1:

**Pull request merged**